src/views/Doctor.vue

```diff
@@ -46,6 +46,17 @@ export default {
   components: {
     Card,
   },
+  methods : {
+    async fetchRecords() {
+    const res = await fetch("https://ban-iot.herokuapp.com/api/health", );
+
+    const data = await res.json()
+    return data      // {"GET"}
+    },
+    async patient () {
```

Comment:
You don’t need to create an extra method.  You could easily set patients in the fetchRecord rather than have another method do that.